Neos.Neos/Classes/Routing/FrontendNodeRoutePartHandler.php

```diff
@@ -579,7 +579,7 @@ protected function buildContextFromWorkspaceNameAndDimensions($workspaceName, ar
     {
         $contextProperties = [
             'workspaceName' => $workspaceName,
-            'invisibleContentShown' => ($workspaceName !== 'live'),
+            'invisibleContentShown' => true,
```

Comment:
This is safe IMHO, since the result of the found node is still subject to handling in the type converter. At least in my tests hidden nodes were not accessible for anonymous requests.

Reply:
absolutely, the type converter saves us there :+1: